spec/spec_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'rspec'
+require_relative '../book'
+require_relative '../person'
+require_relative '../teacher'
+require_relative '../student'
+require_relative '../rental'
+require_relative '../classroom'
+require_relative '../nameable'
+require_relative '../base_decorator'
+# require_relative '../input_handler'
```

Comment:
- [ ] Please remove commented and unused lines of code as they make your codebase look cluttered and difficult to read 👍